addons/popochiu/engine/interfaces/i_inventory.gd

```diff
@@ -143,6 +143,17 @@ func get_item_instance(item_name: String) -> PopochiuInventoryItem:
 	return null
 
 
+## Gets the instance of the [PopochiuInventoryItem] identified with [param script_name].
+func get_inventory_item_instance(script_name: String) -> PopochiuInventoryItem:
+	for rp in PopochiuResources.get_section("inventory_items"):
```

Comment:
Commenting here, but that's valid for each of these getters in the interfaces.

Are you sure you can't `find()` or `filter()` the element instead of looping through the whole array? I think a native implementation would be more readable and performant.

Consider this an open question, I'm not sure it's feasible.

Reply:
Done.

Looks much better now. I sometimes forget I made some functions that can make those search unnecesary.